tests/__snapshots__/CacheTest__it_can_detect_when_the_cache_is_missed__1.json

```diff
@@ -6,9 +6,8 @@
                 "type": "table",
                 "content": {
                     "values": {
-                        "Event": "<code>Hit<\/code>",
-                        "Key": "cached-key",
-                        "Value": "cached-value"
+                        "Event": "<code>Missed<\/code>",
+                        "Key": "cached-key"
```

Comment:
Correctly handle the assertion as it was incorrectly showing `Hit` when it should have been `Missed` due to defaulting to `file` cache driver.